aip/0158.md

```diff
@@ -45,7 +45,7 @@ message ListBooksRequest {
 
 // The response structure from listing books.
 message ListBooksResponse {
-  // The books located on the applicable shelf.
+  // The books from the applicable publisher.
```

Comment:
Ditto earlier comment. (Possibly worth searching for all occurrences of "applicable". Maybe not in this PR though.)

Reply:
```suggestion
  // The books from the specified publisher.
```